packages/@n8n/nodes-langchain/nodes/trigger/ChatTrigger/templates.ts

```diff
@@ -0,0 +1,80 @@
+export function createPage({
+	instanceId,
+	webhookUrl,
+	showWelcomeScreen,
+	i18n: { en },
+	initialMessages,
+	authentication,
+}: {
+	instanceId: string;
+	webhookUrl?: string;
+	showWelcomeScreen?: boolean;
+	i18n: {
+		en: Record<string, string>;
+	};
+	initialMessages: string[];
+	mode: 'test' | 'production';
+	authentication: 'none' | 'basicAuth' | 'n8nUserAuth';
+}) {
+	return `<doctype html>
+	<html lang="en">
+		<head>
+			<meta charset="utf-8">
+			<meta name="viewport" content="width=device-width, initial-scale=1">
+			<title>Chat</title>
+			<link href="https://cdn.jsdelivr.net/npm/normalize.css@8.0.1/normalize.min.css" rel="stylesheet" />
+			<link href="https://cdn.jsdelivr.net/npm/@n8n/chat/style.css" rel="stylesheet" />
+		</head>
+		<body>
+			<script type="module">
+				import { createChat } from 'https://cdn.jsdelivr.net/npm/@n8n/chat@latest/chat.bundle.es.js';
+
+				(async function () {
+					const authentication = '${authentication}';
```

Comment:
This allows XSS since we can just enter any arbitrary HTML into the authentication parameter:
![CleanShot 2024-01-04 at 17 03 27](https://github.com/n8n-io/n8n/assets/12657221/d7fdcdc5-4e61-417f-9099-91963302cecf)
![CleanShot 2024-01-04 at 17 03 42](https://github.com/n8n-io/n8n/assets/12657221/0a0dec79-3ad3-4789-9987-98ff49e5aa9a)

Reply:
😱 Great catch! Sanitized.